docs/tutorial/part-zero/index.md

```diff
@@ -10,48 +10,72 @@ Before you start building your first Gatsby site, you’ll need to familiarize y
+   1. If that fails, download it [directly from Apple's site](https://developer.apple.com/download/more/), after signing-in with Apple developer account
 1. After being prompted to start the installation, you'll be prompted again to accept a software license for the tools to download.
 
-## ⌚ Install Node.js and npm
+## Install Node.js and npm
 
-Node.js is an environment that can run JavaScript code outside of a web browser. Gatsby is built with Node.js. To get up and running with Gatsby, you’ll need to have a recent version installed on your computer.
+Node.js is an environment that can run JavaScript code outside of a web browser. Gatsby is built with Node.js. To get up and running with Gatsby, you’ll need to have a recent version installed on your computer. npm comes bundled with Node.js so if you don't have npm, chances are that you don't have Node.js too.
 
-_Note: Gatsby's minimum supported Node.js version is Node 8, but feel free to use a more recent version._
+### Install Node.js for your appropriate operating system.
 
-1. Open your Terminal.
-1. Run `brew update` to make sure you have the latest version of Homebrew.
-1. Run this command to install Node and npm in one go: `brew install node`
+- On **MacOS** with homebrew installed, run `brew install node`. If you don't want to install it through homebrew, download the latest Node.js version from [the official Node.js website](https://nodejs.org/en/), double click on the downloaded file and go through the installation process.
 
-Once you have followed the installation steps, make sure everything was installed properly:
+- On **Windows**, download and install the latest Node.js version from [the official Node.js website](https://nodejs.org/en/)
 
-### Check your Node.js installation
+- On **Linux**:
 
-1.  Open up your terminal.
-2.  Run `node --version`. (If you’re new to the command line, “run `command`” means “type `node --version` in the command prompt, and hit the Enter key”. From here on, this is what we mean by “run `command`”).
-3.  Run `npm --version`.
+  Install curl and nvm (Node Version Manager). nvm is used to manage node and all its associated versions.
+
+  - Ubuntu, Debian, and other `apt` based distros:
+    - Run `sudo apt-get install curl` to install curl which allows you to transfer data and download additional dependencies.
+    - After it finishes installing, run `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash` to download the latest nvm version.
+    - To confirm this has worked, use the following command. `nvm --version`. The output should be a version number.
+  - Arch, Manjaro and other `pacman` based distros:
+    - These distros come installed with curl, so you can use that to download nvm.
+      `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash`
+    - Before using nvm, you need to install additional dependencies by running `sudo pacman -S grep awk tar git`.
```

Comment:
```suggestion
3. Before using nvm, you need to install additional dependencies by running `sudo pacman -S grep awk tar`.
```
Removing git here since we note that separately below.